ui/admin/app/components/form/credential-library/index.hbs

```diff
@@ -74,6 +74,24 @@
     {{/each}}
   </form.radioGroup>
 
+  <form.fieldset>
```

Comment:
Since radio group has been updated to use `inline` and fieldset labeling, previous radiogroup definition be deleted as part of cleanup. Otherwise UI shows two sets of radiobutton groups right now.

Reply:
sorry forgot to delete the previous one. updated now.

Reply:
No worries at all. Looking good after update.